vips/vip-400/bscmainnet.ts

```diff
@@ -0,0 +1,184 @@
+
+const vip400 = () => {
+  const meta = {
+    version: "v2",
+    title: "vip400 arbitrum Prime configuration",
+    description: `#### Description
+    This VIP will grant permission to timelocks and performs the necessary configuration of OmnichainProposalSender on BNB chain and OmnichainProposalExecutor`,
+    forDescription: "I agree that Venus Protocol should proceed with this proposal",
+    againstDescription: "I do not think that Venus Protocol should proceed with this proposal",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds or not",
+  };
+  return makeProposal(
+    [
+      {
+        target: ARBITRUM_PRIME,
+        signature: "addMarket(address,address,uint256,uint256)",
+        params: [
+          ARBITRUM_COMPTROLLER_CORE,
+          ARBITRUM_VWBTC_CORE,
+          ethers.utils.parseEther("2"),
+          ethers.utils.parseEther("4"),
+        ],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ARBITRUM_PRIME,
+        signature: "addMarket(address,address,uint256,uint256)",
+        params: [
+          ARBITRUM_COMPTROLLER_CORE,
+          ARBITRUM_VUSDC_CORE,
+          ethers.utils.parseEther("2"),
+          ethers.utils.parseEther("4"),
+        ],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ARBITRUM_PRIME,
+        signature: "addMarket(address,address,uint256,uint256)",
+        params: [
+          ARBITRUM_COMPTROLLER_CORE,
+          ARBITRUM_VUSDT_CORE,
+          ethers.utils.parseEther("2"),
+          ethers.utils.parseEther("4"),
+        ],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ARBITRUM_PRIME,
+        signature: "addMarket(address,address,uint256,uint256)",
+        params: [
+          ARBITRUM_COMPTROLLER_LST,
+          ARBITRUM_VWETH_LST,
+          ethers.utils.parseEther("2"),
+          ethers.utils.parseEther("4"),
+        ],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ARBITRUM_PRIME,
+        signature: "setLimit(uint256,uint256)",
+        params: [
+          0, // irrevocable
+          500, // revocable
+        ],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [CONVERTER_NETWORK, "addTokenConverter(address)", arbitrumone.NORMAL_TIMELOCK],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [CONVERTER_NETWORK, "removeTokenConverter(address)", arbitrumone.NORMAL_TIMELOCK],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [ARBITRUM_XVS_VAULT_TREASURY, "fundXVSVault(uint256)", arbitrumone.NORMAL_TIMELOCK],
+        dstChainId: LzChainId.arbitrumone,
+      },
```

Comment:
Should be available to fast track and critical timelocks as well

Reply:
https://github.com/VenusProtocol/vips/pull/429/commits/61a7f6fe6187c0dca945f029779a38cbc6589c00